app/views/layouts/application.html.erb

```diff
@@ -115,5 +115,10 @@
   <%= javascript_include_tag :attachinary %>
   <%= yield :charts_data %>
   <%= cloudinary_js_config %>
+
+  <script type="text/javascript">
```

Comment:
It's necessary to set the i18n current locale in javascript.

https://github.com/fnando/i18n-js#setting-up